index.html

```diff
@@ -0,0 +1,277 @@
+                    <div class="line2"></div>
+                    <p>As the main venue for new media art production in 
+                        Korea, Nabi promotes cross-disciplinary collaboration</p>
+                    </div>
+                </div>
+            </div>
+
+
+            <div class="about-box">
+                <div class="speaker">
+                    <div class="image">
+                    <img src="images/image-5.png" alt="">
+                    </div>
+                    <div class="speak">
+                    <h2>Lila Tretikov</h2>
+                    <p><em>Executive Director of the Wikimedia Foundation</em></p>
+                    <div class="line2"></div>
+                    <p>Lila Tretikov is the Executive Director of the Wikimedia
+                        Foundation, the organization that operates 
+                        Wikipedia.</p>
+                    </div>
+                </div>
+
+            
+                <div class="speaker">
+                    <div class="image">
+                    <img src="images/image-2.png" alt="">
+                    </div>
+                    <div class="speak">
+                    <h2>Kilman Chon</h2>
+                    <div class="line2"></div>
+                    <p>Kilman Chon helped bring the Internet to Asia and
+                        he is an outspoken advocate for the open web and 
+                        digital commons in 2012, he was inducted into the 
+                        inaugural class of the Internet</p>
+                    </div>
+                </div>
+            </div>
+
+            <div class="about-box">
+                <div class="speaker">
+                    <div class="image">
+                    <img src="images/image-4.png" alt="pic">
+                    </div>
+                    <div class="speak">
+                    <h2>Julia Leda</h2>
+                    <p><em>President of Young Pirates of Europe</em></p>
+                    <div class="line2"></div>
+                    <p>European ingetration, political democracy and
+                        participation of youth through online as her major
+                        condern, Reda's report outlining potentials</p>
+                    </div>
+                </div>
+
+                <div class="speaker">
+                    <div class="image">
+                    <img src="images/image-6.png" alt="pic">
+                    </div>
+                    <div class="speak">
+                    <h2>Ryan Dyrud</h2>
+                    <p><em>Berkman Professor of Enterpreneurial Legal Studies</em></p>
+                    <div class="line2"></div>
+                    <p>Benkler studies commons-based peer production, and 
+                        published his seminal book The Wealth of Network in 
+                        2016</p>
+                    </div>
+                </div>
+            </div> -->
```

Comment:
- [ ] Kindly remove those comment lines because there are making your code to not be clean and readable.

Reply:
Thanks, Victor i really appreciate your feedback
i have done exactly what you requested of me